nginx/constants.go

```diff
@@ -0,0 +1,11 @@
+package nginx
+
+const (
+	NGINX      = "nginx"
+	Dependency = NGINX // NOTE: alias for old constant name
```

Comment:
We've tended to remove these aliases when rewriting the buildpack because we now have the name of the dependency documented in the [README](https://github.com/paketo-buildpacks/nginx/blob/master/README.md).